index.js

```diff
@@ -3,15 +3,50 @@
 const werelogs = require('werelogs');
 
 const runServer = require('./lib/api/BackbeatServer');
+const { initManagement } = require('./lib/management/index');
 
-const Config = process.env.TEST_SWITCH !== '1' ? require('./conf/Config') :
-    require('./tests/config.json');
+const testIsOn = process.env.TEST_SWITCH === '1';
+const config = testIsOn ?
+    require('./tests/config.json') : require('./conf/Config');
 
 const Logger = werelogs.Logger;
+const log = new Logger('BackbeatServer:index');
 
 werelogs.configure({
-    level: Config.log.logLevel,
-    dump: Config.log.dumpLevel,
+    level: config.log.logLevel,
+    dump: config.log.dumpLevel,
 });
 
-runServer(Config, Logger);
+function initAndStart() {
+    const repConfig = config.extensions.replication;
+    const sourceConfig = repConfig.source;
+    initManagement({
+        serviceName: 'replication',
+        serviceAccount: sourceConfig.auth.account,
+    }, error => {
+        if (error) {
+            log.error('could not load management db', {
+                error: error.message,
+            });
+            setTimeout(initAndStart, 5000);
+            return;
+        }
+        log.info('management init done');
+
+        const bootstrapList = config.getBootstrapList();
+        repConfig.destination.bootstrapList = bootstrapList;
+
+        config.on('bootstrap-list-update', () => {
+            repConfig.destination = config.getBootstrapList();
```

Comment:
should be `repConfig.destination.bootstrapList = config.getBootstrapList()`

Reply:
nice catch!